src/main/java/com/PBA/authservice/persistance/model/dtos/LoginDto.java

```diff
@@ -0,0 +1,15 @@
+package com.pba.authservice.persistance.model.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class LoginDto {
```

Comment:
Remove this DTO, you need only the token as string on response. You have the endpoint for getting active user.